docs/inspec_and_friends.md

```diff
@@ -40,6 +40,14 @@ control "sshd-11" do
 end
 ```
 
+That said, InSpec is not RSpec. InSpec does not support arbitrary RSpec features. Some key differences:
+
+ * RSpec `describe` blocks are intended to be nested. In InSpec, `describe` 
+ blocks should not be nested; instead use `control` blocks to describe a 
+ higher-level grouping of tests.
+ * RSpec is a tool designed for software engineers. It thus supports a very large range of matchers, to enable testing of software engineering constructs such as exceptions, Object Oriented Programming relationships, and so on. In contrast, InSpec is aimed at compliance practioners and infrastructure testers, so our focus is providing a few, well-supported, easy-to-use universal matchers, such as `cmp`.
```

Comment:
Also we do not want to have a matcher per resource, since it requires us to split logic that belongs together. We are not accepting new matcher without a very good reason.

Reply:
While I agree with this sentiment, I don't know that it needs specific call-out, especially in this section. I think it might be confusing since this document is focused on the differences rather than a declaration of what types of changes we'll accept moving forward.

Reply:
> ... so our focus is providing a few, well-supported, easy-to-use universal matchers, such as `cmp`.
Particular in a contrasting document it feels like you may be saying that RSpec is not. Which I don't believe is your intention.

A smaller set of universal matchers that are focused on readability by a wider audience. This is again where a code sample may help convey the story you are trying to tell.

Reply:
With a focus on matchers if feels like a link to matchers content would be appropriate.